.github/workflows/ci.yml

```diff
@@ -414,11 +420,19 @@ jobs:
           override: true
 
       - name: build
+        if: ${{ needs.env.outputs.new-release == 'true' }}
```

Comment:
Could we just run this on all PR's rather than only running it on release PRs? I know that they are similar tasks, but the extra coverage could be nice.

Reply:
> the build that cargo package performs is the same as cargo build but even more strict

I agree with that interpretation, but I think my lesson from this is that I really don't understand the differences 😅 

In which case it's nice to have to two different build steps, but anything that slips through should be very visible, so I'm not too worried about it.